tags.go

```diff
@@ -30,3 +30,22 @@ func GetTagValue(tags []string, key string, separator string) string {
 
 	return ""
 }
+
+// Add adds a new tag at the beginning of the comma-separated tag list
+func (tags Tags) Add(newTag string) Tags {
+	return Tags(strings.Join(append(tags.Split(), newTag), ", "))
+}
+
+// RemoveFromListByKey removes the first item found from the tag list that contains the specified key
+func (tags Tags) RemoveFromListByKey(key string) Tags {
+	tagsList := tags.Split()
+
+	for i, tag := range tagsList {
+		if strings.Contains(tag, key) {
```

Comment:
Shouldn't it be an exact match e.g. `tag == key` since we are dealing with keys?

Reply:
It shouldn't be an exact match, the tag will be something like `default_billing_address:{id}` and I want to remove the tag that contains `default_billing_address` 🙂

Reply:
Hopefully we won't have multiple similar ones 🙈

Reply:
We shouldn't 🙂 In case, I could remove the `break` statement so duplicates are removed as well